### DISCUSSION TITLE
Portainer and nginx installation

### DISCUSSION THREAD

**Discussion by SOG-web** [postalserver/postal#2114] [Help with using Postal]
Please I have a server managed with portianer and Nginx manager.
Can anyone please put me through how to deploy postal on this server

**Comment by willpower232** [comment]
If you refer to the template docker-compose file you can see that you need 5 containers with different commands https://github.com/postalserver/install/blob/main/templates/docker-compose.yml and then you need to point your nginx at the web container.
